src/lambda_funcs/mod_item/modItem_lambda.py

```diff
@@ -3,11 +3,11 @@
 import psycopg2
 from psycopg2 import Error, sql
 import logging
-import copy
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+#Connects to the database

```

Comment:
The comments are fairly useless.
As a general rule don’t comment when the code is obvious. If its complex and your names don’t make sense then comment.